CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -124,6 +124,16 @@
     typedef UInt8                   UTF8Char;
 #endif
 
+/*
+ * The Swift compiler special cases Boolean on Darwin, this is necessary to avoid
+ * compile warnings.
+ */
+#if CF_BUILDING_CF || !DEPLOYMENT_RUNTIME_SWIFT
+typedef Boolean SWIFT_BOOL;
+#else
+typedef _Bool SWIFT_BOOL;
+#endif
```

Comment:
> On 3 Jan 2016, at 7:05 AM, Philippe Hausler <notifications@github.com> wrote:
> 
> I wonder if there is a better way to do this; I am worried that there will be a size differential in that _Bool may end up being different than Boolean and we could have corruption because of that. For DEPLOYMENT_RUNTIME_SWIFT targets it might be interesting to find the root import of Boolean and conditionally exclude that to define a CFBool similarly to how we define a CFIndex etc.
> 


Have pushed an alternate fix. I don’t really understand what’s going on with DarwinBoolean so maybe there’s still a cleaner approach.